Code/Framework/AzQtComponents/AzQtComponents/Components/Widgets/CardHeader.cpp

```diff
@@ -186,20 +218,45 @@ namespace AzQtComponents
     void CardHeader::setIcon(const QIcon& icon)
     {
         m_icon = icon;
+        updateIconLabel();
+    }
 
-        if (!icon.isNull())
+    void CardHeader::setIconOverlay(const QIcon& iconOverlay)
+    {
+        m_iconOverlay = iconOverlay;
+        updateIconLabel();
+    }
+
+    void CardHeader::setIconClickable(bool clickable)
+    {
+        m_iconLabel->SetClickable(clickable);
+    }
+
+    bool CardHeader::isIconClickable() const
+    {
+        return m_iconLabel->GetClickable();
+    }
+
+    void CardHeader::updateIconLabel()
```

Comment:
It'd be good to get someone who worked on AzQtComponents to review these changes since this'll affect all card headers.

Reply:
@cgalvan, @AMZN-daimini or @alexmontAmazon could you take a look at these changes if you get a chance?

Reply:
Reviewed!